frontend/website/pages/hardware-wallet.mdx

```diff
@@ -12,20 +13,67 @@ Note: macOS and Windows are not officially supported at this time.
 
 ## Installation
 
-**Disclaimer**: This is still highly experimental -- use cautiously, as these instructions may change. Once the hardware wallet app is available on [Ledger Live](https://www.ledger.com/ledger-live), the installation steps will be stabilized.
+**Disclaimer**: This is still highly experimental -- use cautiously, as these
+instructions may change. Once the hardware wallet app is available on
+[Ledger Live](https://www.ledger.com/ledger-live), the installation steps will be stabilized.
 
 The following debian packages are needed to use hardware wallets on linux:
 ```
 sudo apt install libusb-1.0-0-dev libudev1 libudev-dev
 ```
 
+### Make sure your computer is recognising the Ledger device
+
+- Plug in the Ledger device, unlock it with your passcode, and go to the menu. Don't open any app,
+- Run `lsusb`, and see if the device is listed -- it will have ID `2c97:0001`
+- If the Ledger device is not listed, follow the suggestions
+  [here](https://support.ledger.com/hc/en-us/articles/115005165269-Fix-connection-issues).
+
+### Build/download the Coda Ledger application
+
+Either download the precompiled app binary from [here](...), or follow the instructions
+at the end of this document to build the app from source.
+
+### Install the python app loader
+
+The Ledger python loader exists [here](https://github.com/LedgerHQ/blue-loader-python)
+and can be downloaded/installed with the following:
+```
+sudo apt install -y libudev-dev libusb
+pip3 install ledgerblue
+```
+
+After installing, when running on Linux, make sure the following rules have been
+added to `/etc/udev/rules.d/` (e.g. with `sudo vim /etc/udev/rules.d/20-hw1.rules`),
+changing `<UNIX username>` to your unix username:
+
+```
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="2c97", ATTRS{idProduct}=="0000", MODE="0660", TAG+="uaccess", TAG+="udev-acl" OWNER="<UNIX username>"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="2c97", ATTRS{idProduct}=="0001", MODE="0660", TAG+="uaccess", TAG+="udev-acl" OWNER="<UNIX username>"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="2c97", ATTRS{idProduct}=="0004", MODE="0660", TAG+="uaccess", TAG+="udev-acl" OWNER="<UNIX username>"
+```
+
+To load the precompiled app onto the Ledger device, run the following:
```

Comment:
Are these scripts in git@github.com:CodaProtocol/ledger-coda-app.git?
We might need to move the ` ### Download the Coda Ledger app` step to here?

Reply:
they'll be in the link in the 'Build/download the Coda Ledger application' above, but I'll add that for context here too